hydrax/simulation/deterministic.py

```diff
@@ -180,8 +184,9 @@ def run_interactive(  # noqa: PLR0912, PLR0915
 
             # Step the simulation
             for i in range(sim_steps_per_replan):
-                t = i * mj_model.opt.timestep
-                u = controller.get_action(policy_params, t)
+                _t = i * mj_model.opt.timestep + mj_data.time
+                t = jnp.array([_t])  # (1,)
+                u = jit_get_action(policy_params, t)
```

Comment:
I may be missing something but `mujoco.mj_step(mj_model, mj_data)` will increment `mj_data.time` right? So could be simply something like
```suggestion
                u = jit_get_action(policy_params, mj_data.time)
```

Reply:
You're correct, good catch! As a larger improvement, I've jitted the entire `interp_func` instead of just `get_action` for the deterministic script and just generated the entire sequence of open-loop spline interpolations in one call in my newest commits.